### ISSUE TITLE
CHAdeMO Vehicles, Battery

### ISSUE THREAD

**Issue by NJbubo** [dalathegreat/Battery-Emulator#548]
Hi Daniel,

Is the CHAdeMO code working? Use these code anyone?
I tried but it doesn't work for me.

Thanks

**Comment by nagyrobi** [comment]
Do you use/have a proper CHAdeMO plug??

**Comment by NJbubo** [comment]
Yes, I have.

**Comment by nagyrobi** [comment]
Great! Can you point us in the right direction where did you get one?

**Comment by leomeno** [comment]
Hi,
I also am trying to run the CHAdeMO implementation using Olimex ESP32-EVB, Solax Hybrid inverter and Nissan Leaf. I still don't have a CHAdeMO plug, I've ordered some cheaper from Aliexpress, bud I've already made 3rd order, because the previous 2 weren't shipped so they returned the money. But I've got some unfinished still only control-level (without DC power lines) testbed, connected the inverter to the board and connected the vehicle and CAN signals just using some pins in the vehicle's CHAdeMO  port. I had some problems, so I've made some modifications to the CHADEMO-BATTERY.cpp mostly some debug messages and tried to make commisioning of the communication, but I started with the version 7.5 and now I'm testing the 7.7.dev version, I need to test some more..
